### ISSUE TITLE
Slider Component for Gradient Map

### ISSUE THREAD

**Issue by JasonLin43212** [dhmit/democracy_africa#10]
Create a slider component that goes between the years 1981 - 2018
It should change the state every time you move the slider so that the map can update.
This makes it easier for the user to update the years

- [x] Create a slider

- [x] Style it to look nice

- [x] Have a "play" feature that goes through the years one by one so we can see the change in democracy over time

**Comment by Crista2019** [comment]
- [x] add a gradient key to explain meaning of colors

**Comment by JasonLin43212** [comment]
Play button added in `heat_map_color_legend`

**Comment by ryaanahmed** [comment]
@JasonLin43212 -- are there additional features still in heat_map_color_legend unmerged? 

@Mayowa99 -- is that the branch we were discussing?

Would be good to merge this work in, even if we need to cut some of it, as several mapping sprints might want some of these features.

**Comment by Mayowa99** [comment]
Yes, this is the branch. I was about to merge it, should I continue with that? It looks like the css file I used was removed, so I was just looking to figure out if and where the styling exists now.

**Comment by JasonLin43212** [comment]
@ryaanahmed There is no more unmerged features. I think it is fine to merge it now.

**Comment by ryaanahmed** [comment]
@Mayowa99 @JasonLin43212 -- thanks, both. I'll take a look and merge in what we can salvage. (@Mayowa99 -- particularly bc your group isn't doing something with mapping, no need for you to get bogged down with this clean up work.)

**Comment by Mayowa99** [comment]
Okay, sounds good!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
